src/core_plugins/elasticsearch/index.js

```diff
@@ -15,8 +16,6 @@ const DEFAULT_REQUEST_HEADERS = [ 'authorization' ];
 
 module.exports = function ({ Plugin }) {
   return new Plugin({
-    require: ['kibana'],
```

Comment:
Looks like these changes have been applied. @spalger @Bargs, either of you available to take another look? I would like to get this merged as soon as we can to increase the time it's sitting in master.